include/onnxruntime/core/graph/graph.h

```diff
@@ -101,8 +101,9 @@ class Node {
   /** Gets the Node's Node::Type. */
   Node::Type NodeType() const noexcept;
 
-  /** Gets the function body if the #NodeType is fused, or nullptr if not. */
-  const Function* GetFunctionBody() const noexcept;
+  /** Gets the function body if the #NodeType is fused, or nullptr if not. 
+  @remarks Initialization of function body is deferred till the first time it is used. */
```

Comment:
it could be initialized with an explicit call to InitFunctionBodyForNode(). maybe instead indicate that the function body will be initialized if that has not already been done.